src/main/java/com/googleinterns/zoomtube/servlets/TranscriptServlet.java

```diff
@@ -37,42 +52,122 @@
 public class TranscriptServlet extends HttpServlet {
   private static final String TRANSCRIPT_XML_URL_TEMPLATE =
       "http://video.google.com/timedtext?lang=en&v=";
-  private static final String START_ATTRIBUTE = "start";
-  private static final String DURATION_ATTRIBUTE = "dur";
-  private static final String TEXT_TAG = "text";
-  private static final String TEST_VIDEO_ID = "3ymwOvzhwHs";
+  private static final String ATTRIBUTE_START = "start";
+  private static final String ATTRIBUTE_DURATION = "dur";
```

Comment:
In javascript I think we've been naming attribute constants as `ATTR_BLABLA`. Maybe change to match?

Reply:
Done!